domain.go

```diff
@@ -114,17 +121,23 @@ func getPersonFromList(compareValue string, peopleList []Person) Person {
 	return Person{}
 }
 
-func personAttributesAreEqual(sp, dp Person, attributeMap []AttributeMap) bool {
-	caseSensitivityList := getCaseSensitivitySourceAttributeList(attributeMap)
+func personAttributesAreEqual(sp, dp Person, config AppConfig) bool {
+	caseSensitivityList := getCaseSensitivitySourceAttributeList(config.AttributeMap)
+	equal := true
 	for key, val := range sp.Attributes {
 		if !stringsAreEqual(val, dp.Attributes[key], caseSensitivityList[key]) {
-			log.Printf("Attribute %s not equal for user %s. Case Sensitive: %v, Source: %s, Destination: %s \n",
-				key, sp.CompareValue, caseSensitivityList[key], val, dp.Attributes[key])
-			return false
+			if config.Runtime.Verbosity >= 5 {
```

Comment:
Should this `5` be a const too?

Reply:
Good question. Something like `VerbosityMedium`?